nucleus/test_launch_integration.py

```diff
@@ -0,0 +1,117 @@
+import io
+from typing import Any, Callable, Dict, List, Optional, Type
+
+from PIL import Image, ImageDraw
+from pydantic import BaseModel, Extra, ValidationError
+
+# From scaleapi/server/src/lib/select/api/types.ts
+# These classes specify how user models must pass output to Launch + Nucleus.
+
+
+class BoxGeometryModel(BaseModel, extra=Extra.forbid):
+    x: float
+    y: float
+    width: float
+    height: float
+
+
+class BoxAnnotationModel(BaseModel, extra=Extra.forbid):
+    geometry: BoxGeometryModel
+    type: str
+    label: Optional[str] = None
```

Comment:
TODO these keys aren't complete, I'm not entirely sure which ones it makes sense to include